### PR TITLE
Revert "Temporarily change jdk8 openj9 back to using vs2010 (#2227)"

### PR DESCRIPTION
**Pull request by sxa**
Reinstantes https://github.com/AdoptOpenJDK/openjdk-build/pull/2191.

Interestingly we reverted this to do 8u272-b01 but it didn't break the OpenJ9 nightlies, which suggests it may not be as tightly coupled to the other PRs as was initially suggested via https://github.com/AdoptOpenJDK/openjdk-build/pull/2191#issuecomment-718780513

FYI @pshipton since we've been buiding with 2010 again for the last week or so

### CODE REVIEW

**Comment by pshipton** [comment]
I don't see any Windows build in the recent nightlies to look at. I believe the VS2010 builds continue to work as long as they can find msvcp100.dll on the machines. The builds are configured to include msvcp120.dll into the build.

**Comment by pshipton** [comment]
Downloaded a recent nightly from Jenkins. There is no msvcp dll included in the build.  In working builds for VS2013 (such as the Nov 10th build), msvcp120.dll is included.

**Comment by sxa** [comment]
Thanks for the update - we'll put it back up to VS2013 as soon as this PR has approval again. Just surprised me to find that it seemed to still be working with 2010 :-)

**Review by karianna — APPROVED** [review]
LGTM

**Review by M-Davies — APPROVED** [review]

**Comment by sxa** [comment]
Will not be merging this until https://github.com/AdoptOpenJDK/openjdk-build/issues/2255 is resolved

**Pull request merged**
